pkg/oc/cli/cmd/process.go

```diff
@@ -303,38 +369,52 @@ func RunProcess(f kcmdutil.Factory, in io.Reader, out, errout io.Writer, cmd *co
 		}
 	}
 
-	outputFormat := kcmdutil.GetFlagString(cmd, "output")
-	if outputFormat == "describe" {
+	if o.outputFormat == "describe" {
 		if s, err := (&describe.TemplateDescriber{
 			MetadataAccessor: meta.NewAccessor(),
 			ObjectTyper:      legacyscheme.Scheme,
 			ObjectDescriber:  nil,
 		}).DescribeTemplate(resultObj); err != nil {
 			return fmt.Errorf("error describing %q: %v\n", obj.Name, err)
 		} else {
-			_, err := fmt.Fprintf(out, s)
+			fmt.Fprintf(o.ErrOut, "%s\n", "DEPRECATED: --output=describe is deprecated and will be removed in the future")
```

Comment:
> @deads2k added deprecation notice for --output=describe

Pull this for now.  This may count as something we have to keep.  The need is pretty clear.